### DISCUSSION TITLE
Way to import beatmaps and skins on startup

### DISCUSSION THREAD

**Discussion by Tiednine-Dash** [ppy/osu#30900] [Ideas]
Basically, like a directory where you can place files like .osz and .osk that'll be scanned when you start the client, and if any files are found, they'll be imported. Similar to how you can place .osz files in the songs directory in stable and have them be imported the next time you start the game up.

**Comment by peppy** [comment]
If you're okay with some DIY work, you could make a batch script which does this for you, by checking the directory and specifying the files as parameters when starting the game.

**Reply by Tiednine-Dash** [reply]
I created one and it turned out to work great. Thanks for the suggestion.
